CONTRIBUTING.md

```diff
@@ -92,6 +92,13 @@ To use `rubocop`, perform the bundle install with no exclusions
     bundle install --path .bundle --with test
     bundle exec rake rubocop
 
+### Run puppetfile install
+
+To install bundled content, run:
+```
```

Comment:
The section below shows the same command. In https://github.com/puppetlabs/bolt/pull/1121/commits/2ba079dfa4afeb0cd0fa4ffc533deef0530d94de I noted that some modules are necessary for testing.